### PR TITLE
i18n(zh-cn): use <Steps> components for many files.

### PR DESCRIPTION
**Pull request by huyikai**
#### Description (required)

use <Steps> components for many files (https://github.com/withastro/docs/pull/8167)

update middleware,netlify

#### Related issues & labels (optional)

- #8167 
- #8176 
- #8178

### CODE REVIEW

File: src/content/docs/zh-cn/guides/cms/buttercms.mdx [hunk 1]
```diff
@@ -27,11 +27,13 @@ import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
 
 ### 配置
 
+<Steps>
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/cms/buttercms.mdx:30]: 这个页面的步骤显示不太对。

File: src/content/docs/zh-cn/guides/cms/strapi.mdx [hunk 2]
```diff
@@ -146,83 +146,83 @@ export default interface Article {
...
+    const articles = await fetchApi<Article[]>({
+      endpoint: 'articles', // 需要获取的内容类型
+      wrappedByKey: 'data', // 在响应中被拆封的键
+    });
+    ---
+    ```
+
+    该 API 调用请求会从 `http://localhost:1337/api/articles` 获取数据，并返回 `articles`，这是一个表示数据的 JSON 对象数组：
+
+    ```
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/cms/strapi.mdx:167]:
```suggestion
    ```json
```

**Review by liruifengv — COMMENTED** [review]
CMS section reviewed

**Review by liruifengv — COMMENTED** [review]
deploy section reviewed

**Review by liruifengv — COMMENTED** [review]
All reviewed!

**Comment by liruifengv** [comment]
middleware  页面有更新： https://github.com/withastro/docs/pull/8176

**Comment by liruifengv** [comment]
netlify: https://github.com/withastro/docs/pull/8178

**Comment by huyikai** [comment]
> netlify: #8178
middleware,netlify 已同步

**Comment by liruifengv** [comment]
这又有一个更新： https://github.com/withastro/docs/pull/7218

**Review by liruifengv — APPROVED** [review]
LGTM! Good Job!

**Pull request merged**